### ISSUE TITLE
PostWRF installation problems (Arch Linux)

### ISSUE THREAD

**Issue by Nikolaidis8416** [anikfal/PostWRF#20]
Hello, it's my first time with PostWRF, I use Arch Linux and when I try to install it happened it:

![Problema_PostWRF](https://github.com/anikfal/PostWRF/assets/20303246/1948199a-35d5-4e96-b2bf-ab713f25dcd3)

Please, I need help with it.

**Comment by anikfal** [comment]
Greetings,

You need to set a ssh-key in your local machine and add it to your github profile. You can follow the instructions bellow:
1) [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent?platform=linux)
2) [Adding a new SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account?tool=webui&platform=linux)

----
In summary:
- `cd ~/.ssh`
- `ssh-keygen`
It will save a new file (`id_rsa.pub`) in `~/.ssh` - You can set no password.
- Copy the contents of `id_rsa.pub`
- Go to Github `Settings` -> `SSH and GPG keys` -> `New SSH key`
- Paste the contents of `id_rsa.pub`

Now you must be able to clone any repositories into your local machine.
